DevelopmentTools/HCALLMapDumper.cc

```diff
@@ -652,23 +652,27 @@ void HCALLMapDumper::printHOCalibLMapObject(std::vector<HOCalib> myHOCalib)
 {
   std::cout << "#Dumping HO Calib LMap Object..." << std::endl;
   std::cout << "#"
-            << std::setw(6) << "Side" << std::setw(6) << "Eta" << std::setw(6) << "Phi" << std::setw(6) << "dPhi" << std::setw(6) << "Depth" << std::setw(10) << "Det"
+            << std::setw(6) << "Side" << std::setw(6) << "Eta" << std::setw(6) << "Phi" << std::setw(6) << "dPhi" << std::setw(9) << "ch_type" << std::setw(12) << "name" << std::setw(6) << "det"
             << std::setw(9) << "RBX"
             << std::setw(6) << "Sect"
             << std::setw(6) << "RM" << std::setw(6) << "RM_FI" << std::setw(6) << "FI_CH"
-            << std::setw(6) << "ppCol" << std::setw(6) << "ppRow" << std::setw(6) << "ppLC"
-            << std::setw(6) << "Crate" << std::setw(6) << "HTR" << std::setw(6) << "HTRTB" << std::setw(9) << "HTR_FI"
+            << std::setw(6) << "ppCol" << std::setw(6) << "ppRow" << std::setw(20) << "ppCpl" << std::setw(6) << "ppLC"
+            << std::setw(6) << "Crate" << std::setw(6) << "HTR" << std::setw(6) << "fpga" << std::setw(9) << "HTR_FI"
+            << std::setw(9) << "QIEid"
+            << std::setw(6) << "dcc" << std::setw(8) << "spigot"
```

Comment:
Please move dcc spigot to the location after line crate, but before QIEid. Because those variables belong to backend